src/doc.c

```diff
@@ -2325,6 +2327,41 @@ void highlightText(Scope *sc, Dsymbol *s, OutBuffer *buf, size_t offset)
                 }
                 break;
 
+            case '`':
+                if (inBacktick)
+                {
+                    inBacktick = 0;
+                    inCode = 0;
+
+                    OutBuffer codebuf;
+
+                    codebuf.write(buf->data + iCodeStart + 1, i - (iCodeStart + 1));
+                    codebuf.writeByte(0);
+
+                    highlightCode2(sc, s, &codebuf, 0);
+
+                    buf->remove(iCodeStart, i - iCodeStart + 1); // also trimming off the current `
+                    i = buf->insert(iCodeStart, codebuf.data, codebuf.offset);
+                    i = buf->insert(i, ")", 1);
+
+                    break;
+                }
+
+                if (inCode)
+                    break;
+
+                inCode = 1;
+                inBacktick = 1;
+
+                static const char pre[] = "$(D_INLINECODE ";
```

Comment:
BTW if inline colored code is ever needed, a builtin macro should be defined for it. But that's orthogonal to the backquote's job. Backqouted text is not D-specific at all; it's up to the user to make it so.